src/vs/editor/browser/controller/coreCommands.ts

```diff
@@ -1698,11 +1692,19 @@ class EditorOrNativeTextInputCommand extends Command {
 			return;
 		}
 
+		let editorService: any = accessor.get(IEditorService);
+		let activeEditor = editorService.getActiveEditor && editorService.getActiveEditor();
+
+		if (activeEditor.isWebviewEditor && activeEditor.webview) {
+			activeEditor.webview.selectAll();
+			return;
+		}
```

Comment:
Looks like this was just for testing? I don't think `EditorOrNativeTextInputCommand` should know anything about `selectAll`

Reply:
Yes, this was unintentional. I also found a better solution in general so I will be pushing that up when I iron out the kinks.